src/Estimators/include/BipedalLocomotion/RobotDynamicsEstimator/SubModelCreator.h

```diff
@@ -0,0 +1,176 @@
+/**
+ * @file SubModelCreator.h
+ * @authors Ines Sorrentino
+ * @copyright 2023 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the BSD-3-Clause license.
+ */
+
+#ifndef SUBMODELCREATOR_H
+#define SUBMODELCREATOR_H
+
+#include <vector>
+#include <string>
+#include <memory>
+
+// iDynTree
+#include <iDynTree/Model/Model.h>
+#include <iDynTree/ModelIO/ModelLoader.h>
+#include <iDynTree/KinDynComputations.h>
+
+// BLF
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+
+namespace BipedalLocomotion
+{
+namespace Estimators
+{
+namespace RobotDynamicsEstimator
+{
+
+
+/**
+ * Sensor describes the generic sensors of the submodels.
+ */
+struct Sensor
+{
+    std::string name; /**< Name of the sensor in the URDF */
+    std::string frame; /**< Frame of the sensor in the URDF */
+};
+
+/**
+ * FT describes the force/torque sensors of the submodels.
+ */
+struct FT : Sensor
+{
+    enum class Direction : int { positive=1, negative=-1, notspecified=0 };
```

Comment:
I realized that we don't have a specific code style for the components of an ``enum class``, although the most common style we used is with the first letter capitol and then camel case. @GiulioRomualdi what do you think?